config.ts

```diff
@@ -1,6 +1,12 @@
 import { defineConfig } from "./lib/configMeta.ts";
 
 export default defineConfig({
+	antennaCalibration: {
+		distance_0: 5.0,
+		strength_0: -22.0,
+		distance_1: 15.0,
+		strength_1: -38.0,
```

Comment:
I don't like just adding _num at the end of names. Especially not when all other names are camelCase. Is it possible to do something like maybe `first.distance`and `second.distance` ?